omise/omise-php
# Add Receipt API

### Summary
- [x] Add Receipt API ([Omise doc](https://www.omise.co/receipts-api))

### QA
You should be able to do the following actions.

- List receipts
`$receipts = OmiseReceipt::retrieve();`

- Retrieve a receipt
`$receipt = OmiseReceipt::retrieve('rcpt_54jvtzr5pwwbeks22ae');`

- Reload a receipt
`$receipt->reload();`
